integration/jnifuse/fs/src/main/java/alluxio/jnifuse/AbstractFuseFileSystem.java

```diff
@@ -327,6 +327,11 @@ public int removexattrCallback(String path, String name) {
     return 0;
   }
 
+  public int symlinkCallback(String linkname, String path) {
+    LOG.debug("Ignore unsupported symlink operation, linkname {}, path{}", linkname, path);
```

Comment:
If we don't implement this function, thinking about return an error code instead of swallow the error only left a line of log.

Reply:
I have modified it. how do you feel the current way~